plugins/vocabbuilder.koplugin/db.lua

```diff
@@ -217,21 +217,19 @@ function VocabularyBuilder:gotOrForgot(item, isGot)
     local target_review_count = math.max(item.review_count + (isGot and 1 or -1), 0)
     local target_count = isGot and item.streak_count + 1 or 0
     if target_count == 0 then
-        due_time = current_time + 5 * 60
+        due_time = current_time + 5 * 60 -- 5 minutes
     elseif target_count == 1 then
-        due_time = current_time + 30 * 60
+        due_time = current_time + 24 * 3600 -- 24h
```

Comment:
I'd like to retain the 30min and 12h checks to strengthen short-term memory.

Reply:
Updated.